quickstart.sh

```diff
@@ -28,6 +31,10 @@ do
 			offline_lab=true
       log_major "Running Chorus with offline lab environment enabled"
 			;;
+    --apple-silicon | -apple)
+      apple_silicon=true
+      log_major "Configuring Chorus for Apple Silicon"
+      ;;
```

Comment:
I know mysql was fixed without needing this flag. Docker should just work to determine the right image to pull.

Reply:
The fewer flags to the `quickstart.sh`, the better!   It's meant for a newbie to quickly use, so we don't want too many options...  every option just makes it more confusing....    Maybe someday we'll have a `powerstart.sh`  script that does all the things we would want if Chorus was actually a project that could be deployed in a production like environment versus a learning tool!